.github/actions/push-to-ecr/action.yml

```diff
@@ -0,0 +1,78 @@
+name: Push images to the production ECR
+
+inputs:
+  image_name:
+    description: 'The name of the image (for example "aica-technology/image-name")'
+    required: true
+    type: string
+  image_tags:
+    description: 'The tags of the image (for example "latest,v1.0.0")'
+    required: true
+    type: string
+  archs:
+    description: 'The archs to merge as comma-separated list (for example "amd64,arm64")'
+    required: true
+    type: string
+  token:
+    description: "The GitHub token passed from the caller workflow"
+    required: true
+    type: string
+  ecr_name:
+    description: "The name of the target ECR"
+    required: true
+    type: string
+
+runs:
+  using: composite
+  steps:
+    - name: Create symlink for actions from this repository
+      run: |
+        ln -s ${{ github.action_path }}/../.. .gha-gmm
+      shell: bash
+
+    - uses: ./.gha-gmm/actions/login-to-ghcr
+      with:
+        token: ${{ inputs.token }}
+
+    - uses: ./.gha-gmm/actions/ghcr-ensure-prefix
+      id: ensure-image
+      with:
+        image_name: ${{ inputs.image_name }}
+
+    - uses: ./.gha-gmm/actions/list-add-suffixes
+      id: calculate-images
+      with:
+        list: ${{ steps.ensure-image.outputs.image_name }}
+        suffixes: ${{ inputs.image_tags }}
+        glue_separator: ":"
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v3
+      with:
+        role-to-assume: arn:aws:iam::336280353617:role/github-actions-role
```

Comment:
Do you mean to store these values in github? Or make these values to inputs and the user needs to define something like
```
env:
  DAY_OF_WEEK: Monday
```
in order to use the action?

Reply:
It would be repeated all over no?

I guess the role would be fair game if we were to have one for each repo (which is probably best practice here) but the rest (ECR and region) I think we can define globally. They aren't sensitive, just config.